CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 2.2.0-nullsafety
+
+Pre-release for the null safety migration of this package.
+
+Note that 2.2.0 may not be the final stable null safety release version, we
```

Comment:
I'm SO GLAD we're being clear here. Thanks!

CC @jakemac53

Reply:
I just copy/pasted this from Jake's changelogs. Thanks Jake!